tests/bluetooth/common/testlib/CMakeLists.txt

```diff
@@ -27,3 +27,21 @@ target_sources(testlib PRIVATE
 target_sources_ifdef(CONFIG_BT_SMP testlib PRIVATE
   src/security.c
 )
+
+# Helpers that can be used on native_posix targets but don't have other
+# dependencies (e.g. on Bluetooth, etc).
+add_library(bs_helpers)
```

Comment:
Shouldn't this be a `zephyr_library` and all cmake calls below the `zephyr_` equivalents?

Reply:
I'm not completely sure what effect of `zephyr_library` is. Using CMake standard `add_library` hides less stuff, so it's simpler to understand what's happening in a build, I think. Explicit is better than implicit, right?